Source/DafnyLanguageServer.Test/Various/StartLanguageServer.cs

```diff
@@ -0,0 +1,11 @@
+// using System;
```

Comment:
Why is this file commented out?

Reply:
My bad. It's a development relic. Deleting..